libpdbg/target.h

```diff
@@ -83,6 +83,18 @@ struct pdbg_target_class *require_target_class(const char *name);
 struct pdbg_target_class *get_target_class(struct pdbg_target *target);
 bool pdbg_target_is_class(struct pdbg_target *target, const char *class);
 
+/**
```

Comment:
why this isn;t static void function in device.c

Reply:
I don't know. Not my changes. My changes are only at LN87